compiler-rt/lib/builtins/cpu_model/AArch64CPUFeatures.inc

```diff
@@ -33,7 +33,6 @@ enum CPUFeatures {
   FEAT_FP,
   FEAT_SIMD,
   FEAT_CRC,
-  FEAT_SHA1,
```

Comment:
Are there ABI stability concerns if you modify this enum?

Reply:
It was raised on another pr, so perhaps it is.

Reply:
This is an ABI break, but FMV still very much in beta status so we can make changes to the ABI. My understanding is that compiler-rt is not required to be compatible with versions of clang/llvm other than the one it ships with (please correct me if that's not the case). And the clang and gcc implementations of FMV are already ABI incompatible for other reasons.

Reply:
> And the clang and gcc implementations of FMV are already ABI incompatible for other reasons.
Are they? I'm not currently aware of any incompatibilities.

Reply:
We've generally been more conservative with compiler-rt.builtins compared with other parts of compiler-rt, like  sanitizers.  People do ship static libraries to other people, and we don't want them to silently break.

Even if you still consider the ABI "beta", it's not clear to me how we expect users to know; is the "beta" status mentioned anywhere in LLVM/clang documentation?